lib/src/api/err/mod.rs

```diff
@@ -39,30 +34,33 @@ pub enum Error {
 	AlreadyConnected,
 
 	/// `Query::bind` not called with an object nor a key/value tuple
-	#[error("Invalid bindings: {0}")]
+	#[error("Invalid bindings: {0:?}")]
 	InvalidBindings(Value),
```

Comment:
Semantically shouldn't we avoid exposing `Debug` to a user message? Can't we replay on `Display`?

Reply:
I changed them back!